contracts/proxy/ManagerProxy.sol

```diff
@@ -29,13 +29,27 @@ contract ManagerProxy is ManagerProxyTarget {
         targetContractId = _targetContractId;
     }
 
+    /**
+     * @notice Fallback function that delegates calls to target contract when there is no msg.data
+     */
+    receive() external payable {
+        _fallback();
+    }
+
+    /**
+     * @notice Fallback function that delegates calls to target contract when there is msg.data
+     */
+    fallback() external payable {
+        _fallback();
+    }
+
     /**
```

Comment:
For consistency let's switch from notice tag -> dev tag here as was done in https://github.com/livepeer/protocol/pull/538/files#diff-9195324af6678c2d259508c144349c83e0bb45cb1fe3c4fb14c28e5f0a0f9e47R45 since this is a private function and not user facing?